ModularTegustation/tegu_mobs/syndicatespaceman.dm

```diff
@@ -0,0 +1,33 @@
+/mob/living/simple_animal/hostile/syndicate/melee/sword/space/oldcode
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "oldcode_syndicate_csaber"
+	icon_living = "oldcode_syndicate_csaber"
+	name = "Syndicate Spaceman"
+	desc = "Death to IS-Nanotrasen."
+	maxHealth = 170
+	health = 170
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	armour_penetration = 20
+	light_color = LIGHT_COLOR_BLUE
+	minbodytemp = 0
+	speed = 1
+	var/obj/effect/light_emitter/blue_energy_sword/
```

Comment:
It's never used, from what I can see in the code. Check how normal syndi NPCs use it.

Reply:
as far as i can tell the caravan syndie uses COLOR_SOFT_RED instead of LIGHT_COLOR_BLUE and uses red_energy_sword/sord rather than var/obj/effect/light_emitter/blue_energy_sword/. It seems to be working, kinda

![Screenshot_508](https://user-images.githubusercontent.com/58402542/108403137-3f5c8c00-71d3-11eb-8c91-d74991d9f1a0.png)
old caravan simplemob
![Screenshot_507](https://user-images.githubusercontent.com/58402542/108403173-497e8a80-71d3-11eb-8bcb-b6cc98fe8637.png)
oldcode simplemob